test/replication-luatest/gh_9720_tmp_spaces_ddl_abort_applier_txs_test.lua

```diff
@@ -0,0 +1,166 @@
+local t = require('luatest')
+local replica_set = require('luatest.replica_set')
+local server = require('luatest.server')
+
+local g = t.group(nil, t.helpers.matrix({memtx_use_mvcc = {false, true}}))
+
+g.before_each(function(cg)
+    cg.replica_set = replica_set:new{}
+    local box_cfg = {
+        memtx_use_mvcc_engine = cg.params.memtx_use_mvcc,
+        replication = {
+            server.build_listen_uri('server1', cg.replica_set.id),
+            server.build_listen_uri('server2', cg.replica_set.id),
+        },
+        replication_timeout = 0.1,
+    }
+    cg.server1 = cg.replica_set:build_and_add_server{
+        alias = 'server1',
+        box_cfg = box_cfg,
+    }
+    box_cfg.read_only = true
+    cg.server2 = cg.replica_set:build_and_add_server{
+        alias = 'server2',
+        box_cfg = box_cfg,
+    }
+    cg.replica_set:start()
+    cg.server1:exec(function()
+        local s = box.schema.create_space('s')
+        s:create_index('p')
+        box.begin()
+        for i = 1, 1000 do
+            s:replace{i, i}
+        end
+        box.commit()
```

Comment:
Why do you need so many tuples in the space? Same below in `test_remote_ddl()`.

Reply:
Because otherwise I won't get to the error injection:
https://github.com/tarantool/tarantool/blob/09bfdd473ffd23169dbe8d4e5d0510e8c70b11ec/src/box/memtx_space.c#L52-L65